src/Perl6/Pod.nqp

```diff
@@ -1,25 +1,33 @@
 class Perl6::Pod {
 
     # various helper methods for Pod parsing and processing
-    my $caption := ''; # var to save table caption values between 
+    my $caption := ''; # var to save table caption values between
                        # subs make_config and table
 
+    # hashes for quoting pairs in config semilist values
+    # pairs of stand-alone quoting chars
+    my %h  := nqp::hash(
+        #'<',    '>', # angle brackets (shouldn't be in a semilist)
+        "'",     "'", # single quote
+        '"',     '"', # double quote
+    );
+
+    # pairs of quoting chars after a leading Q or q (this may not be an exaustive list)
+    my %h2 := nqp::hash(
+        '<',     '>', # angle brackets
+        "'",     "'", # single quote
+        '"',     '"', # double quote
+        '|',     '|', # pipe
+        '[',     ']', # square brackets
+        '{',     '}', # curly braces
+    );
+
     # enable use of env vars for debug selections
-    # TODO: track down possible nqp bug: inconsistent handling of the debug values
-    #       see possible solution in World.nqp, grep getenvhash
-    my $debug    := 0; # for dev use
-    my $udebug   := 0; # for users via an environment variable
-    my $ddenvvar := 'RAKUDO_POD6_TABLE_DEBUG_DEV';
-    my $duenvvar := 'RAKUDO_POD6_TABLE_DEBUG';
-    my %env      := nqp::getenvhash();
-    if nqp::existskey(%env, $ddenvvar) {
-        my $val := nqp::atkey(%env, $ddenvvar);
-        $debug := $val;
-    }
-    if nqp::existskey(%env, $duenvvar) {
-        my $val := nqp::atkey(%env, $duenvvar);
-        $udebug := $val;
-    }
+    # for users
+    my $udebug := nqp::ifnull(  nqp::atkey(  nqp::getenvhash(), 'RAKUDO_POD6_TABLE_DEBUG'  ), 0  );
```

Comment:
I wonder, why is it “POD6” sometimes and just “POD” in other cases?

Reply:
um, not paying attention, which do you suggest? my original thought for users it should be clear it’s not p5 pod. then when i started working on more than table pod i thought just POD is clear enough for contributers.

what should be used?